python/humbug/report.py

```diff
@@ -35,19 +44,30 @@ class Modes(Enum):
     SYNCHRONOUS = 1
 
 
+def make_request(method: str, url: str, **kwargs):
```

Comment:
Good idea for this kind of function to be a `Reporter` method. That allows you to share HTTP connections on a per-reporter basis.

Reply:
(e.g. in the future you can use a [`requests.Session`](https://docs.python-requests.org/en/master/user/advanced/#session-objects) to maintain a single HTTP connection per reporter)

Reply:
Now just send request and open connection for each request. https://bugout.atlassian.net/browse/BUG-401